slurm/entrypoint.sh

```diff
@@ -68,21 +77,23 @@ then
     done
     echo "-- slurmctld is now active ..."
 
-    echo "---> Starting sshd on the slurmd..."
-    /usr/sbin/sshd -e
-
     echo "---> Starting the Slurm Node Daemon (slurmd) ..."
     exec /usr/sbin/slurmd -Dv
 fi
 
 if [ "$1" = "frontend" ]
 then
     echo "---> Starting SSSD ..."
+    # Sometimes on shutdown pid still exists, so delete it
+    rm -f /var/run/sssd.pid
     /sbin/sssd --logger=stderr -d 3 -i 2>&1 &
 
     echo "---> Starting the MUNGE Authentication service (munged) ..."
     gosu munge /usr/sbin/munged
 
+    echo "---> Starting sshd on the frontend..."
+    /usr/sbin/sshd -D -e
```

Comment:
This might not work, because sshd was last thing to start it was what kept frontend container running.  That's also why it has `-D` flag. I think for frontend sshd needs to remain last.

Reply:
Didnt notice that one,  will revert.